### DISCUSSION TITLE
Welcome to millfork Discussions!

### DISCUSSION THREAD

**Discussion by KarolS** [KarolS/millfork#91] [General]
##  Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zbyti** [comment]
Hi :]

I'm simple 8-bit chess enthusiast and I hoping to write my own engine in the future so I'm always curious how efficient is given 8-bit compiler, because of that I wrote benchmark suite:

https://github.com/zbyti/a8-mad-pascal-bench-suite

Thanks to @markjfisher we have C version of my code:

https://github.com/markjfisher/kickc-benchmarks

I'll do my best to create Millfork version of the suite, which (I hope) help us to keep track progression or regression in speed department.

**Comment by agg23** [comment]
I'm a retro computing enthusiast with a focus on early Macintosh and Nintendo consoles. Millfork has served as the home of much of my NES game experience, and I hope to continue contributing to Millfork as situations arrive. My current focus on Millfork itself is #81, implementing language server support (and thus enabling an easier development experience).

**Comment by james7780** [comment]
Hi, I'm a retro homebrew programmer, mostly on Atari Lynx, Atari 5200 and MSX/Spectravideo, so looking to use Millfork for writing games and tools on these platforms.
Right now I'm busy creating a graphics library/module and audio library/module for the Lynx in MF, however converting my existing C code to MF is going pretty slow (mostly due to the difference in the way pointers work, specially pointer to an element in an array).
I'm also looking at creating a config .ini for SVI 328 (Spectravideo).
